packages/testing/src/random.ts

```diff
@@ -0,0 +1,11 @@
+import { randomBytes } from 'node:crypto';
+
+import type { Address } from 'viem';
+import { isAddress, keccak256 } from 'viem/utils';
+
+export function generateRandomEthAddress(): Address {
```

Comment:
you're already using viem. this is much simpler:
```
import { privateKeyToAccount, generatePrivateKey } from 'viem/accounts';

export function generateWallet() {
  const key = generatePrivateKey();

  const account = privateKeyToAccount(key);
  }
  ```

Reply:
In some cases, I just need an ETH address. When generating a private key, it adds a small performance hit